src/Markdig/Extensions/Tables/PipeTableParser.cs

```diff
@@ -59,7 +61,7 @@ public override bool Match(InlineProcessor processor, ref StringSlice slice)
             var position = processor.GetSourcePosition(slice.Start, out int globalLineIndex, out int column);
             var localLineIndex = globalLineIndex - processor.LineIndex;
 
-            if (tableState == null)
+            if (tableState is null)
```

Comment:
I don't mind these changes (personally prefer is/is not), but it doesn't make a difference for nullable analyzers right?

Reply:
The analyzers pick up both patterns, but is is slightly more correct as it bypasses the equality operator and prevents use on structs.  I'd like to followup with another PR after this to replace the remaining == null with is null / for consistency.